README.md

```diff
@@ -157,6 +157,13 @@ fn main() {
     window.show();
 
     event_loop.run_forever(|_event| {
+        match _event {
+            glutin::Event::WindowEvent { event, .. } => match event {
+                glutin::WindowEvent::Closed => return glutin::ControlFlow::Break,
```

Comment:
If you only have one match, the `return` is not necessary.

Reply:
Sorry but I can not fix it. I just started learning Rust. Rust By Example, or Book. Doesn't help here.